Withings/OAuth1/DefaultTokenManager.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Withings.API.Portable.OAuth1
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    class DefaultTokenManager : ITokenManager //from ITokenManager
```

Comment:
In our 1.0 library for Fitbit we didn't have a TokenManager... you might find you don't need one here since OAuth 1.0a doesn't refresh tokens. But we can circle back on this as you get farther into a working sample of device connection.